components/observatorium.libsonnet

```
@@ -29,7 +28,7 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
         deduplicationReplicaLabels: obs.config.deduplicationReplicaLabels,
         deleteDelay: '48h',
       },
-    },
+    } + (if std.objectHas(obs.config.compact, 'enableDownsampling') && obs.config.compact.enableDownsampling == true then {} else t.compact.withDownsamplingDisabled),
```

I think a new line would be nice here. At least something like:
```suggestion
} + (
  if std.objectHas(obs.config.compact, 'enableDownsampling') && obs.config.compact.enableDownsampling == true then
    {}
  else t.compact.withDownsamplingDisabled
)
```

---

done